rest-api/tools/setup_env.sh

```diff
@@ -11,7 +11,7 @@ find . | grep \.pyc | xargs rm -if $*
 
 echo "Installing libs..."
 # If this fails due to missing mysql_config, try `sudo apt-get install libmysqlclient-dev`.
-pip install -r requirements.txt -t lib/
+pip install --system -r requirements.txt -t lib/
```

Comment:
Did the default (no `--system` or `--user`) stop working here?

Reply:
Yeah, it didn't work for me when I tried running it (with or without --user). Not sure why.